index.html

```diff
@@ -22,39 +22,58 @@
             <form class="navbar-form inline pull-right">
                 <a href="https://wiki.mozilla.org/Telemetry" class="btn" title="Wiki" target="_blank">Telemetry Wiki</a>
```

Comment:
Why is Telemetry Wiki special enough to merit its own top-of-the-page link? Would it do better under Documentation, or is it a mishmash of different things?

And a personal opinion: I dislike target="_blank" links, because it's not evident they're going to act differently, and I rarely desire that behavior.

Reply:
Ah, I see Frank says we can remove it completely. Spiffy.